torch/csrc/Module.cpp

```diff
@@ -1244,11 +1244,7 @@ class WeakTensorRef {
   }
 };
 
-extern "C"
-#ifdef _WIN32
-    __declspec(dllexport)
-#endif
-        TORCH_API PyObject* initModule();
+extern "C" C10_EXPORT PyObject* initModule();
```

Comment:
Shouldn't it be `TORCH_EXPORT`?

Reply:
No such a macro defined, only TORCH_API

Reply:
No macro called TORCH_EXPORT. Only C10_EXPORT is defined to be __declspec(dllexport). It makes little sense to define TORCH_EXPORT to point to C10_EXPORT for this code snippet.